biofefi/utils/utils.py

```diff
@@ -125,4 +125,18 @@ def delete_directory(path: Path):
     Args:
         path: The path to the log directory
     """
-    shutil.rmtree(path, ignore_errors=True)
+    if path.is_dir():
+        shutil.rmtree(path, ignore_errors=True)  # Remove the directory
+
+
+def delete_file(path: Path):
+    """Delete the previous results from the log_dir
```

Comment:
This function is about deleting a file generally, not just results in a log_dir :)